server/channels/api4/post.go

```diff
@@ -1107,6 +1110,76 @@ func unacknowledgePost(c *Context, w http.ResponseWriter, r *http.Request) {
 	ReturnStatusOK(w)
 }
 
+func moveThread(c *Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
Possibly. Maybe not in this iteration, but a future one? 

Wrangler's functionality will be severely hampered if in its current state we add another layer of permissions on top of it. I think that it would be best to address this in later iterations as ideally in the future the "config" of this plugin will move more towards the built in permissions of Mattermost _instead of_ rather than _in addition to_.

Reply:
LGTM, but then I guess it is more a question to product and security :P